saw-remote-api/python/saw/__init__.py

```diff
@@ -52,6 +53,16 @@ class VerificationResult(metaclass=ABCMeta):
     @abstractmethod
     def is_success(self) -> bool: ...
 
+class ProofResult(metaclass=ABCMeta):
+    goal: proofscript.ProofScript
+    valid: bool
+    counterexample: Any
+
+    def is_valid(self) -> bool:
+        return self.valid
+
+    def get_counterexample(self) -> Any:
+        return self.counterexample
```

Comment:
method docstring (in particular, what is returned if the `ProofResult` `is_valid`?)